definitions/features/downstream.rb

```diff
@@ -74,6 +74,13 @@ def rh_repos(sat_version)
     rh_repos
   end
 
+  def ansible_repo(sat_version)
+    if sat_version.to_s == '6.4' || sat_version.to_s == '6.5'
```

Comment:
As we already have sat_version i.e instance of `Version` class, you can modify this comparison to `sat_version >= version('6.4')`. 
So that we don't need to modify this condition everytime, it will consider all sat version >= `6.4`.